datasafe-simple-adapter/datasafe-simple-adapter-spring/src/test/java/de/adorsys/datasafe/simple/adapter/spring/InjectionTest.java

```diff
@@ -32,4 +60,50 @@ public void testCreateUser(SimpleDatasafeService datasafeService) {
         assertThat(datasafeService.userExists(userid)).isTrue();
         datasafeService.destroyUser(userIDAuth);
     }
+
+    @SneakyThrows
+    void testWithoutPathEncryption(SimpleDatasafeService simpleDatasafeServiceApi, DFSCredentials dfsCredentials) {
+        if (!(simpleDatasafeServiceApi instanceof SimpleDatasafeServiceImpl)) {
+            throw new RuntimeException("Did expect instance of SimpleDatasafeServiceImpl");
+        }
+        AbsoluteLocation<PrivateResource> rootLocation = getPrivateResourceAbsoluteLocation(dfsCredentials);
+        SimpleDatasafeServiceImpl simpleDatasafeService = (SimpleDatasafeServiceImpl) simpleDatasafeServiceApi;
+
+        UserIDAuth userIDAuth = new UserIDAuth(new UserID("peter"), ReadKeyPasswordTestFactory.getForString("password"));
+        String content = "content of document";
+        String path = "a/b/c.txt";
+        DSDocument document = new DSDocument(new DocumentFQN(path), new DocumentContent(content.getBytes()));
+        simpleDatasafeService.createUser(userIDAuth);
+        simpleDatasafeService.storeDocument(userIDAuth, document);
+
+        try (Stream<AbsoluteLocation<ResolvedResource>> absoluteLocationStream = simpleDatasafeService.getStorageService().list(rootLocation).filter(el -> el.location().toASCIIString().contains(path))) {
+            Assertions.assertEquals(1, absoluteLocationStream.count());
+        }
+        try (Stream<AbsoluteLocation<ResolvedResource>> absoluteLocationStream = simpleDatasafeService.getStorageService().list(rootLocation).filter(el -> el.location().toASCIIString().contains(path))) {
```

Comment:
It is better to move out `filter` clause int try-with-resources body, as the derived stream might not close the originating stream

Reply:
I though, try with resource is bullet proof and I dont see why filter keeps stream open. However, I did changes. Hope this is correct now.